build.gradle

```
@@ -20,7 +20,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
+        classpath 'com.android.tools.build:gradle:7.0.4'
```

Seems not related to this issue.

```suggestion
        classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
```